eftcamb/07p1_Pure_EFT_std.f90

```diff
@@ -145,13 +149,21 @@ subroutine EFTCAMBPureEFTstdAllocateModelSelection( self )
                 write(*,'(a,I3)') 'No model corresponding to PureEFTmodelOmega =', self%PureEFTmodelOmega
                 write(*,'(a)')    'Please select an appropriate model.'
         end select
-        ! allocate wDE:  IW: need to add the other parametrizations
+        ! allocate wDE: 
         if ( allocated(self%PureEFTwDE) ) deallocate(self%PureEFTwDE)
         select case ( self%EFTwDE )
             case(0)
                 allocate( wDE_LCDM_parametrization_1D::self%PureEFTwDE )
             case(1)
                 allocate( constant_parametrization_1D::self%PureEFTwDE )
+            case(2)
+                allocate( CPL_parametrization_1D::self%PureEFTwDE )
```

Comment:
We can (hopefully) put here the call to parametrized_function_1D%set_param_names to change the names of the parameters of the functions!